libs/types/src/test_utils.ts

```diff
@@ -0,0 +1,14 @@
+import { assertDefined } from '@votingworks/basics';
+import { Election, SplittablePrecinct } from './election';
+
+// istanbul ignore next - @preserve
+export function generateSplittablePrecinctsForTest(
+  sourceElection: Election
+): SplittablePrecinct[] {
+  return sourceElection.precincts.map(
+    (p): SplittablePrecinct => ({
+      ...p,
+      districtIds: [assertDefined(sourceElection.districts[0]).id],
```

Comment:
For this to be technically correct, we'd need to go through the ballot styles to find the district list for each precinct and create splits accordingly. Not sure how important it is for the tests that are using this to have the correct precincts and splits

Reply:
Currently it's not important because we're only using it to unlock compatibility where tests call existing translation functions that expect `SplittablePrecincts` to be passed in.

String extraction itself is tested here: https://github.com/votingworks/vxsuite/pull/5991/files#diff-2e159f5332cddf7352b9c7a86bbde3761202da561cacf3198bc921fd4242a58cR133